apicula.dockerfile

```diff
@@ -0,0 +1,46 @@
+# Authors:
+#   Unai Martinez-Corral
+#   Lucas Teske
+#
+# Copyright 2019-2021 Unai Martinez-Corral <unai.martinezcorral@ehu.eus>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+FROM hdlc/build:build AS build
+
+RUN apt-get update -qq \
+ && DEBIAN_FRONTEND=noninteractive apt-get -y install --no-install-recommends wget
+
+RUN mkdir /opt/apicula \
+ && wget https://files.pythonhosted.org/packages/1a/d6/b3162f87ff114d639095fe7c0655080ee16caff9037d6629f738d8b28d92/Apycula-0.0.1a6.tar.gz \
```

Comment:
Are nightly builds available? That would allow us to track a fixed tarball name, instead of having to bump it manually.

In any case, please set `0.0.1a6` as a variable, for reducing the maintenance burden.

Reply:
I dont think it makes much difference to put `0.0.1a6` in variable since that hash on the URL changes as well (basically the entire URL changes).

Reply:
`0.0.1a6` is used not only in the URL, but also in the call to `tar` (previously, when calling `mv` and `rm` too). It's ok not to use a variable if the version number is shown in the URL only. Then, please add `-o apycula.tgz` to `wget`, so that the next `tar` call is invariant.